examples/docs/en-US/form.md

```diff
@@ -838,6 +838,7 @@ All components in a Form inherit their `size` attribute from that Form. Similarl
 | label-position | position of label | string | left / right / top | right |
 | label-width | width of label, and all its direct child form items will inherit this value | string | — | — |
 | label-suffix | suffix of the label | string | — | — |
+| required-asterisk | whether required fields should have a red asterisk (star) beside their labels | boolean | — | true |
```

Comment:
The new attribute can be named as `hide-required-asterisk` which default value is `false`.

Reply:
Done!